src/main/java/mate/academy/bookproject/model/Book.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.bookproject.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name = "book")

```

Comment:
don't use `@Data` for entities
table name should be plural according to the naming convention
```suggestion
@Getter
@Setter
@Table(name = "books")
```